cmd/mapt/cmd/aws/hosts/fedora.go

```diff
@@ -46,7 +47,8 @@ func getFedoraCreate() *cobra.Command {
 			if err := viper.BindPFlags(cmd.Flags()); err != nil {
 				return err
 			}
-			return fedora.Create(
+			mCtx, err := maptContext.Init(
```

Comment:
:thinking: but why do you need to pass this as param?  ...can you init the context inside the context.Init function?